docs/question/overview.md

```diff
@@ -12,6 +12,17 @@ A question is made up of three main components:
 2. **`question.html`**: The HTML template that defines the question. This is where you write the question text and define the input elements. Reference the [question template documentation](template.md) for more details.
 3. **`server.py`**: This is where you write the logic for generating random values, grading student responses, and any other server-side code. This file is optional, but it is necessary for any question that has non-trivial randomization or custom grading behavior. Reference the [server documentation](server.md) for more details.
 
+## Creating a question
```

Comment:
Nit: I wonder if it's better to move this section above the "Question components" section. Having the new section here breaks the flow between the two sections that are now before and after this point.